plugins/inputs/github/README.md

```diff
@@ -0,0 +1,45 @@
+# GitHub Input Plugin
+
+The [GitHub](https://www.github.com) input plugin gathers statistics from GitHub repositories.
+
+### Configuration:
+
+```toml
+[[inputs.github]]
+    repositories = [
+      "owner/repository",
+    ]
+
+    ## Optional: Unauthenticated requests are limited to 60 per hour
+    access_token = "personal access token"
```

Comment:
We have some information on [SampleConfig](https://github.com/influxdata/telegraf/wiki/SampleConfig) style on the wiki.  Use a 2 space indent and comment out any optional settings showing the default value.  No need to list them as optional or specify the default in the comment.